code/modules/shuttle/mini_dropship.dm

```diff
@@ -273,3 +280,40 @@
 	origin.shuttle_port.set_mode(SHUTTLE_CALL)
 	origin.last_valid_ground_port = origin.my_port
 	SSshuttle.moveShuttleToDock(origin.shuttleId, origin.my_port, TRUE)
+
+/obj/machinery/computer/camera_advanced/shuttle_docker/minidropship/proc/rappel_targeting(datum/source, atom/target, turf/location)
```

Comment:
This proc should be modularized to the actual object and not to the camera computer